supervisord/setup.conf

```diff
@@ -0,0 +1,2 @@
+[program:setup-wp]
```

Comment:
does this config replaces previous one?

Reply:
It does not replace the wp-setup.php - it is to run the wp install and wp plugin install commands